src/test/java/org/ays/auth/controller/AysUserControllerTest.java

```diff
@@ -638,4 +641,94 @@ void givenValidIdAndInvalidUserUpdateRequest_whenRoleIdIsNotValid_thenReturnVali
+
+        //Then
+        String endpoint = BASE_PATH.concat("/user/").concat(mockId).concat("/activate");
+        MockHttpServletRequestBuilder mockHttpServletRequestBuilder =
+                AysMockMvcRequestBuilders.patch(endpoint, mockAdminToken.getAccessToken());
+
+        AysResponse<Void> mockResponse = AysResponseBuilder.SUCCESS;
+
+        aysMockMvc.perform(mockHttpServletRequestBuilder, mockResponse)
+                .andExpect(AysMockResultMatchersBuilders.status().isOk())
+                .andExpect(AysMockResultMatchersBuilders.response().doesNotExist());
+
+        //Verify
+        Mockito.verify(userUpdateService,Mockito.times(1)).activate(Mockito.anyString());
+    }
+
+    @Test
+    void givenValidId_whenUserNotFound_thenReturnAysUserNotExistByIdException() throws Exception {
+        // Given
+        String mockId = AysRandomUtil.generateUUID();
+
+        // When
+        Mockito.doThrow(new AysUserNotExistByIdException(mockId)).when(userUpdateService).activate(Mockito.anyString());
+
+        // Then
+        String endpoint = BASE_PATH.concat("/user/").concat(mockId).concat("/activate");
+        MockHttpServletRequestBuilder mockHttpServletRequestBuilder = AysMockMvcRequestBuilders
+                .patch(endpoint, mockAdminToken.getAccessToken());
+
+        AysErrorResponse mockErrorResponse = AysErrorBuilder.NOT_FOUND;
+
+        aysMockMvc.perform(mockHttpServletRequestBuilder, mockErrorResponse)
+                .andExpect(AysMockResultMatchersBuilders.status().isNotFound())
+                .andExpect(AysMockResultMatchersBuilders.response().doesNotExist());
+
+        // Verify
+        Mockito.verify(userUpdateService, Mockito.times(1)).activate(Mockito.anyString());
+    }
+
+    @Test
+    void givenValidId_whenUserIsNotPassive_thenReturnAysUserIsNotPassiveException() throws Exception {
+        // Given
+        String mockId = AysRandomUtil.generateUUID();
+
+        // When
+        Mockito.doThrow(new AysUserIsNotPassiveException(AysUserStatus.ACTIVE)).when(userUpdateService).activate(Mockito.anyString());
+
+        // Then
+        String endpoint = BASE_PATH.concat("/user/").concat(mockId).concat("/activate");
+        MockHttpServletRequestBuilder mockHttpServletRequestBuilder = AysMockMvcRequestBuilders
+                .patch(endpoint, mockAdminToken.getAccessToken());
+
+        AysErrorResponse mockErrorResponse = AysErrorBuilder.NOT_FOUND;
+
+        aysMockMvc.perform(mockHttpServletRequestBuilder, mockErrorResponse)
+                .andExpect(AysMockResultMatchersBuilders.status().isNotFound())
+                .andExpect(AysMockResultMatchersBuilders.response().doesNotExist());
+
+        // Verify
+        Mockito.verify(userUpdateService, Mockito.times(1)).activate(Mockito.anyString());
+    }
+
+    @Test
+    void givenValidId_whenUserUnauthorized_thenReturnAccessDeniedException() throws Exception {
+        // Given
+        String mockId = AysRandomUtil.generateUUID();
```

Comment:
Testlerde verileri daha net görebilmek adına aşağıdaki şekilde kullanabiliriz, diğer testlerdeki kullanımlar da yavaş yavaş düzeltilecek
```suggestion
        String mockId = "201aec72-ecd8-49fc-86f5-2b5458871edb";
```